ethclient/ethclient.go

```diff
@@ -188,6 +188,28 @@ func (tx *rpcTransaction) UnmarshalJSON(msg []byte) error {
 	return json.Unmarshal(msg, &tx.txExtraInfo)
 }
 
+// TransactionByHash returns the transaction with the given hash.
+func (ec *Client) TransactionByHashWithBlockNum(ctx context.Context, hash common.Hash) (tx *types.Transaction, blockHexString string, err error) {
+	var json *rpcTransaction
+	err = ec.c.CallContext(ctx, &json, "eth_getTransactionByHash", hash)
```

Comment:
`&json` seems to be a pointer to a pointer

Reply:
The code is exactly copied from the function TransactionByHash and it seems to be using the same there as well.